android/src/main/kotlin/com/posthog/posthog_flutter/PosthogFlutterPlugin.kt

```diff
@@ -29,6 +33,8 @@ class PosthogFlutterPlugin : FlutterPlugin, MethodCallHandler {
         channel.setMethodCallHandler(this)
     }
 
+    private val isLoaded = MutableStateFlow(false);
```

Comment:
I'd not use coroutines but rather implement a dependency-free mechanism, there are a few days to do this.

Reply:
makes sense